modules/cnf-performing-end-to-end-tests-junit-test-output.adoc

```diff
@@ -20,15 +20,18 @@ Use the following procedures to generate a JUnit latency test output and test fa
 +
 [source,terminal,subs="attributes+"]
 ----
-$ podman run -v $(pwd)/:/kubeconfig:Z -v $(pwd)/junitdest:<junit_folder_path> \
--e KUBECONFIG=/kubeconfig/kubeconfig -e DISCOVERY_MODE=true -e FEATURES=performance \
-registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version} \
-/usr/bin/test-run.sh --junit <junit_folder_path> \
--ginkgo.focus="\[performance\]\ Latency\ Test"
+$ podman run -v $(pwd)/:/kubeconfig:Z -v $(pwd)/junit:/junit \
```

Comment:
is this suppose to be the file path to where the `junit` folder is stored? If so, I'd put </junit_file_path>

Reply:
QE has confirmed this is correct as is. `junit` is the actual folder name I think